src/plugins/janus_videoroom.c

```diff
@@ -7103,6 +7103,9 @@ static void janus_videoroom_recorder_create(janus_videoroom_publisher_stream *ps
 					janus_videoroom_media_str(ps->type));
 			}
 		}
+		/* If the stream has a description, store it in the recording */
+		if(ps->description && rc)
```

Comment:
I think the `&& rc` part is unneeded, since the setter can do the `NULL` check internally.

Reply:
Fixed